en/micro-integrator/docs/references/connectors/configure-connector-exporter-project.md

```diff
@@ -0,0 +1,27 @@
+## Configure the connector in WSO2 Integration Studio
+
+1. Open WSO2 Integration Studio and create an ESB Solution Project.
+<p><img src="../../assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" /></p>
+
+2. Right click on the project that you created and click on 'Add or Remove Connector' -> Add Connector.
+<p><img src="../../assets/img/connectors/add-connector.png" title="Add or Remove Connector" width="800" alt="Add or Remove Connector" /></p>
+
+3. It will be navigated to the WSO2 Connector Store. 
+4. Search for File Connector and download it to the workspace. 
+<p><img src="../../assets/img/connectors/search-connector.png" title="Search Connector in the Connector Store" width="800" alt="Search Connector in the Connector Store" /></p>
+
+5. Now it needs to add a Connector Exporter Project. 
+6. Go to File -> New -> Other -> WSO2 -> Extensions -> Project Types -> Connector Exporter Project
+<p><img src="../../assets/img/connectors/connector-exporter-project-1.png" title="Add Connector Exporter Project" width="800" alt="Add Connector Exporter Project" /></p>
+
+<p><img src="../../assets/img/connectors/connector-exporter-project-2.png" title="Add Connector Exporter Project" width="800" alt="Add Connector Exporter Project" /></p>
+
+7. Provide a name for Connector Exporter Project and in the next screen tick on 'Specify the parent from workspace' and select the File Connector project. 
```

Comment:
```suggestion
7. Enter a name for the Connector Exporter Project and in the next screen select **Specify the parent from workspace** and select the File Connector project. 
```

I feel all the list items in this page seem to be too compressed. It's like we are trying to cover multiple steps in one step itself. Shall we rethink this and have one step per list item?

Reply:
Splitted them to two steps.